src/test/cli/ceph-conf/show-config-value.t

```diff
@@ -18,3 +18,18 @@
   mon_host=$public_network
   $mon_host
   $ rm $TESTDIR/ceph.conf
+
+Name option test to strip the PID
+=================================
+  $ echo '[client]' > $TESTDIR/ceph.conf
+  $ echo 'admin socket = /tmp/ceph-asok.rUoNeE/$name.$pid.asok' >> $TESTDIR/ceph.conf
+  $ ceph-conf --name client.admin.133423 --show-config-value admin_socket -c $TESTDIR/ceph.conf | awk -F'/' '{print $NF}'
+  client.admin.133423.asok
```

Comment:
```suggestion
  /.*/client.admin.133423.asok (re)
```

Reply:
Will similify this by changing the `admin socket` to `$name.$pid.asok`